.github/workflows/twister.yaml

```diff
@@ -188,6 +188,7 @@ jobs:
             git log  --pretty=oneline | head -n 10
           fi
           echo "$HOME/.local/bin" >> $GITHUB_PATH
+          echo "$HOME/.cargo/bin" >> $GITHUB_PATH
```

Comment:
I'd move this as well, keep the steps isolated at some extent, you don't need it here, right?

Reply:
It needs to be in a separate run step from the rustup command below. This won't get removed, even if the rustup target add command does.